README.md

```diff
@@ -26,12 +26,23 @@ relative_files = True
 
 ### Local usage
 
+#### Tests from changes in Git working directory
+
 1. Install `pytest-rts` with `pip install pytest-rts`
 2. Create a branch `git checkout -b feat/new-feature`
 3. Make changes in your code
 4. Run the tool with `pytest --rts --rts-coverage-db=[path to database]`
 
-As a result only tests related to changes in working directory and branch will be executed.
+As a result only tests related to changes in working directory will be executed.
+
+#### Tests from working directory + committed changes 
```

Comment:
Let's have the title starting the same way as previous: "Tests from changes in Git working directory + committed changes" to make it clear that it is the same as previous with some extra.